lib/smart_answer/calculators/statutory_sick_pay_calculator.rb

```diff
@@ -131,6 +141,20 @@ def valid_contractual_days_covered_by_earnings?
         false
       end
 
+      def valid_coronavirus_incapacitated_nogp?
```

Comment:
I don't see these methods actually being used anywhere - could they be fixed up to the commit where they're used so that their usefulness is more obvious?

Reply:
These methods are no longer being used and will be removed.